docs/tiers.md

```diff
@@ -0,0 +1,483 @@
+    * `kubernetesSubjectAccessReview`: Uses the SAR API to ask Kubernetes: "Can this user (`auth.identity.user.username`) perform the `post` action on the `llminferenceservices` resource for the model being requested?"
+    * The `namespace` and `name` of the model are dynamically parsed from the request path (e.g., `/model-serving-ns/my-model-name/...`).
+
+4.  **`response.success.identity` (Identity Enrichment):** After all checks pass, this section prepares the identity that will be used by other policies, like `RateLimitPolicy`.
+    * It creates a clean identity object containing the normalized `userid`.
+    * Crucially, it extracts the tier name from the metadata (`auth.metadata.matchedTier["tier"]`) and exposes it as `auth.identity.tier`. This is the field the `RateLimitPolicy` uses in its `when` clause.
+
+**Apply:**
+```bash
+kubectl apply -f gateway-auth-policy.yaml
+```
+
+**For complete auth policy configuration examples**, see the repository files.
+
+---
+
+### Authorization (RBAC)
+
+RBAC controls which tiers (via their namespaces) can access models. This is done by creating a `Role` with model access 
+permissions and a `RoleBinding` to link that `Role` to the tier namespaces.
+
+**Example (for all models in the namespace):**
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: model-user
+  namespace: model-serving
+rules:
+  - apiGroups: ["serving.kserve.io"]
+    resources: ["llminferenceservices"]
+    verbs: ["post"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: model-user-tier-binding
+  namespace: model-serving
+subjects:
+  - kind: Group
+    name: system:serviceaccounts:openshift-ai-inference-tier-free
+    apiGroup: rbac.authorization.k8s.io
+  - kind: Group
+    name: system:serviceaccounts:openshift-ai-inference-tier-premium
+    apiGroup: rbac.authorization.k8s.io
+  - kind: Group
+    name: system:serviceaccounts:openshift-ai-inference-tier-enterprise
+    apiGroup: rbac.authorization.k8s.io
+roleRef:
+  kind: Role
+  name: model-user
+  apiGroup: rbac.authorization.k8s.io
+```
+
+> [!IMPORTANT]
+> **Why only `post` verb?**  
+> The `post` verb grants permission to send inference requests without giving users broader access to the Kubernetes resources. 
+> With only `post`, users **cannot**:
+> - Inspect model configurations (`get`)
+> - Modify resource definitions (`update`, `patch`)
+> - Delete model deployments (`delete`)
+> - Enumerate available models (`list`)
+> 
+> This strict confinement ensures that users can consume the AI service via the gateway without having any direct management access to the underlying resources.
+
+**Apply:**
+```bash
+kubectl apply -f model-rbac.yaml
+```
+
+---
+
+### Rate Limiting
+
+The `RateLimitPolicy` enforces different quotas per tier using the `auth.identity.tier` field that was enriched by the `AuthPolicy`.
+
+**Request-based limits:**
+
+```yaml
```

Comment:
maybe a little description/note that this example applies rate limits to a specific model server but the same can be applied at the Gateway level ?